src/renderer/components/ExperimentalSettings/ExperimentalSettings.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <FtSettingsSection
+    :title="$t('Settings.Experimental Settings.Experimental Settings')"
+  >
+    <p class="experimental-warning">
+      {{ $t('Settings.Experimental Settings.Warning') }}
+    </p>
+    <FtFlexBox>
+      <FtToggleSwitch
+        tooltip-position="top"
+        :label="$t('Settings.Experimental Settings.Replace HTTP Cache')"
+        compact
+        :default-value="replaceHttpCache"
+        :disabled="replaceHttpCacheLoading"
+        :tooltip="$t('Tooltips.Experimental Settings.Replace HTTP Cache')"
+        @change="handleRestartPrompt"
+      />
+    </FtFlexBox>
+    <FtPrompt
+      v-if="showRestartPrompt"
+      :label="$t('Settings[\'The app needs to restart for changes to take effect. Restart and apply change?\']')"
+      :option-names="[$t('Yes, Restart'), $t('Cancel')]"
+      :option-values="['restart', 'cancel']"
+      @click="handleReplaceHttpCache"
+    />
+  </FtSettingsSection>
+</template>
+
+<script setup>
+import { onMounted, ref } from 'vue'
+
+import FtSettingsSection from '../ft-settings-section/ft-settings-section.vue'
+import FtFlexBox from '../ft-flex-box/ft-flex-box.vue'
+import FtToggleSwitch from '../ft-toggle-switch/ft-toggle-switch.vue'
+import FtPrompt from '../ft-prompt/ft-prompt.vue'
+
+import { IpcChannels } from '../../../constants'
+
+const replaceHttpCacheLoading = ref(false)
```

Comment:
Default value changed?
```suggestion
const replaceHttpCacheLoading = ref(true)
```